code/src/components/AddTask.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+
+import { remember } from 'reducers/remember'
+import { RoundedContainer } from 'lib/Container'
+import { Button } from 'lib/Button'
+import { Select } from 'lib/Select'
+
+export const AddTask = () => {
+  const [text, setText] = useState('')
+  const [priority, setPriority] = useState('')
+  const dispatch = useDispatch()
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    dispatch(remember.actions.addTask({ text, priority }))
+    setText('') // clearing input
+    setPriority('') // clearing input
+  }
+
+  return (
+    <RoundedContainer>
+      <form onSubmit={handleSubmit}>
+        <input
+          type="text"
+          value={text}
+          onChange={(event) => setText(event.target.value)}
+          placeholder="add an item"
+          required />
+
+        <label>
+          Set Priority:
```

Comment:
"Set priority" needs a bit margin to the left so it is not so close to the input field.